northstar_tests/tests/tests.rs

```diff
@@ -347,6 +348,50 @@ test!(connect_version, {
     runtime.shutdown().await
 });
 
+test!(connections_during_shutdown_are_ignored, {
+    let runtime = Northstar::launch().await?;
+    let console = runtime.config().console.as_ref().unwrap();
+
+    runtime.install_test_container().await?;
+    runtime.install_test_resource().await?;
+
+    let console_url = console.clone();
+    let start_container = tokio::task::spawn(async move {
+        let name = "test_container";
+        let version: Version = Version::parse("0.0.1").unwrap();
+        loop {
+            match api::client::Client::new(&console_url, None, time::Duration::from_secs(30)).await
+            {
+                Ok(client) => client.start(&name, &version).await.ok(),
+                Err(_) => break,
+            };
+        }
+    });
+
+    let console_url = console.clone();
+    let stop_container = tokio::task::spawn(async move {
+        let name = "test_container";
+        let version: Version = Version::parse("0.0.1").unwrap();
+        loop {
+            match api::client::Client::new(&console_url, None, time::Duration::from_secs(30)).await
+            {
+                Ok(client) => client
+                    .stop(&name, &version, std::time::Duration::from_secs(1))
+                    .await
+                    .ok(),
+                Err(_) => break,
+            };
+        }
+    });
+
+    let result = runtime.shutdown().await;
```

Comment:
This is not true, there is nothing preventing `start_container` and `stop_container` to execute before this shutdown. The current master won't pass this test because connections might be still awaiting replies in the event loop. That's the reason for the explicit drop of `event_rx` in the event loop.

Reply:
Think my text is misleading: I tested with the shutdown of the console moved to the earliest possible place (see comment above). It's very likely that the shutdown  is almost *always* initiated before the two tasks will start to do something. At least I haven't seen a run where either the start or stop task managed even to connect.

Reply:
Added a `logger::assume` to wait for the start of the container before this line just to be sure that the client tasks are executing during shutdown.

Reply:
Cool.